bin/load_env_vars.sh

```diff
@@ -125,6 +125,8 @@ function set_e2e_env_vars() {
   export S3_FORM_ID="$(s3_form_id $environment)"
   export AWS_S3_BUCKET="$(aws_s3_bucket $environment)"
   export SETTINGS__AWS__S3_SUBMISSION_IAM_ROLE_ARN="$(aws_s3_role_arn $environment)"
+  export SETTINGS__SUBMISSION_STATUS_API__SECRET="$(get_param /${environment}/automated-tests/e2e/runner/submission_status_api_shared_secret)"
```

Comment:
The parameter keys for the secrets in environments other than deploy look like

`arn:aws:ssm:eu-west-2:${data.aws_caller_identity.current.account_id}:parameter/forms-runner-${var.environment_name}/submission_status_api_shared_secret`

https://github.com/alphagov/forms-deploy/blob/baa23ab2a438137f1ce2576512863e154a48eb97/infra/modules/code-build-run-e2e-tests/codebuild.tf#L93

Reply:
Actually my comment is wrong... what you've done is correct.

Reply:
thanks for checking @lfdebrux !